pnc-core/src/main/java/org/jboss/pnc/core/builder/BuildCoordinator.java

```diff
@@ -260,7 +257,7 @@ void startBuilding(BuildTask buildTask) throws CoreException {
                 }
                 return buildResult;
             } catch (BuildDriverException e) {
```

Comment:
Yeah, it could be a solution. But if someone make some updates the code and forgets to catch all exceptions, it can happen, but maybe I'm too careful. I'll catch Throwable.